ui/src/components/TopBar/index.tsx

```diff
@@ -83,9 +83,9 @@ export default class Topbar extends React.PureComponent<IProps, IState> {
                             Stats
                         </Button> */}
                         <Button href="/apps">Apps</Button>
-                        <Button href="https://blog.podcastindex.org/">
-                            Blog
-                        </Button>
+                        {/*<Button href="https://blog.podcastindex.org/">*/}
+                        {/*    Blog*/}
```

Comment:
@daveajones Have we abandoned the blog? If so, I am good to merge this.

Reply:
Mostly.  May bring it back, but for now best to remove it.